src/core/public/workspace/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+export { WorkspacesClientContract, WorkspacesClient } from './workspaces_client';
+export { WorkspacesStart, WorkspacesService } from './workspaces_service';
+export { WorkspaceAttribute, WorkspaceFindOptions } from '../../server/types';
```

Comment:
@SuZhou-Joe  these variables should be exported in public level, otherwise there will be an error.
![image](https://github.com/ruanyl/OpenSearch-Dashboards/assets/42465692/407edc08-e8f3-41ec-872f-4e675eba1c1c)